src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,7 +122,16 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        String string = DATE_FORMATTER.format(someDate);
+        LocalDate now = LocalDate.now();
+        if (someDate.isEqual(now)) {
+            string += " is today";
+        } else {
```

Comment:
let's remove this check and if it is not before or after we we consider is todat